src/astro/utils/snowflake_append.py

```diff
@@ -34,11 +34,11 @@ def wrap_identifier(inp):
 
     if columns or casted_columns:
         statement = (
-            "INSERT INTO Identifier(%(main_table)s) ({main_cols}{sep}{main_casted_cols})"
-            "(SELECT {fields}{sep}{casted_fields} FROM Identifier(%(append_table)s))"
+            "INSERT INTO %(main_table)s ({main_cols}{sep}{main_casted_cols})"
```

Comment:
why are we removing - Identifier()? this may lead to SQL injections right?

Reply:
The identifier is actually now being added in in the sql_decorator.py. We now have it set that it can detect tables and automatically do that. We don't have that same detection for columns which is why those are still there.